components/sidebar/sidebar_category_list/sidebar_category_list.tsx

```diff
@@ -216,6 +230,10 @@ export default class SidebarCategoryList extends React.PureComponent<Props, Stat
     }
 
     updateUnreadIndicators = () => {
+        if (this.props.draggingState.state) {
```

Comment:
It might make sense to hide the unread indicators when dragging in addition to not recalculating it.